Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -185,7 +185,7 @@ try {
                             $message = $errorLines.ToString().Trim()
                             $null = $errorLines.Clear()
                             if ($message) {
-                                Write-VstsTaskError -Message $message
+                                Write-VstsTaskError -Message $message -IssueSource $IssueSources.CustomerScript
```

Comment:
[This is only a suggestion -- not blocking or a must do]

I see sometimes we specify "CustomerScript" other times "Internal".  What is the default?

To avoid ambiguity, it might be better to just change the option to one of  "-SourceCustomerScript" or "-SourceInternal"; only provide one, exclusion would mean the other! (that might be would be more self-documenting)

Reply:
The default is "Internal". To implement the logic you've described we should be sure that we have an opportunity to manage stdout from the user script. I'm not sure that it's possible for the PowerShell handler in the current implementation.